### DISCUSSION TITLE
Is it possible to modify commands commans on the music widget (the big one, not the one in the bar)?

### DISCUSSION THREAD

**Discussion by NextStep-IM** [end-4/dots-hyprland#245] [Q&A]
Do the buttons use `playerctl` for next, previous and stop? Now that multiple player windows are available, I want to modify commands on each player window of the widget. But I can't make sense of it. The music widget in the bar is using `playerctl` but I can't find anything for the big widget.  

##### What I wanted to do:
Modify the commands on each player window like this: 
`playerctl --player=player_1 play/pause`
`playerctl --player=player_2 play/pause`

**Comment by end-4** [comment]
stuff in those music controls use AGS' Mpris service
which should work cleanly in theory but it doesn't sometimes... idk
